CMakeLists.txt

```diff
@@ -7,8 +7,8 @@ disallow_intree_builds()
 project (utf8proc C)
 
 # Be sure to also update these in Makefile!
-set(SO_MAJOR 3)
-set(SO_MINOR 0)
+set(SO_MAJOR 2)
```

Comment:
Note that we changed the SO version in #24 because of the `struct` changes in #20.

But maybe we should stop considering the `struct` to be part of the ABI?  Maybe we should stop exposing it in the header file.